edb/pgsql/metaschema.py

```diff
@@ -5011,6 +5015,260 @@ def _generate_schema_alias_view(
     )
 
 
+def _generate_sql_information_schema() -> List[dbops.View]:
+    sql_ident = 'information_schema.sql_identifier'
+    sql_str = 'information_schema.character_data'
+    sql_bool = 'information_schema.yes_or_no'
+    sql_card = 'information_schema.cardinal_number'
+    tables_and_columns = [
+        dbops.View(
+            name=('edgedbsql', 'tables'),
+            query=(
+                f'''
+        WITH obj_ty AS (
+            SELECT
+                id,
+                CASE SPLIT_PART(name, '::', 1)
+                    WHEN 'default' THEN 'public'
+                    ELSE SPLIT_PART(name, '::', 1)
+                END AS schema_name,
+                SPLIT_PART(name, '::', 2) AS table_name
+            FROM edgedb."_SchemaObjectType"
+            WHERE internal IS NOT TRUE
+        ),
+        tables (id, schema_name, table_name) AS ((
+            SELECT * FROM obj_ty
+        ) UNION ALL (
+            WITH qualified_links AS (
+                -- multi links and links with at least one property
+                -- (besides source and target)
+                SELECT link.id
+                FROM edgedb."_SchemaLink" link
+                JOIN edgedb."_SchemaProperty" AS prop ON link.id = prop.source
+                WHERE prop.computable IS NOT TRUE AND prop.internal IS NOT TRUE
+                GROUP BY link.id, link.cardinality
```

Comment:
Logic that seems so simple when stuck in a python function can get a lot gnarlier inlined into a SQL query... :)